.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -55,7 +55,14 @@ example output
 
 ### File(s) scanned
 
-Optionally attach the file (e.g. Bazel profile).
+Optionally provide file(s) that demonstrate the bug when running the tool.
+
+Ideally, do not attach files directly. Bazel profiles can include sensitive data, such as exposing
+which dependencies your project includes.
+
+Instead, upload files to the cloud storage provider of your choice and grant appropriate access.
```

Comment:
What is "appropriate access" (varies by provider of course)? Do we want/can we add a shared engflow user they can share with, assuming most providers allow you to share with a specific email?

Reply:
I consciously left this somewhat vague, as it greatly depends on the use case and the cloud storage provider used (as you already noted).
It may be that some providers allow adding a group, like bazel-invocation-analyzer-dev@engflow.com.
It may also be that sharing via link is sufficient, either by sending that link to just bazel-invocation-analyzer-dev@engflow.com, or by posting it publicly (e.g. in the GitHub issue), but having the option to revoke that access later on.
If we need individual access, we can to begin with just asking for access for the person assigned to the issue. If greater need arises, we can look into a better solution. However, at this point I'd first like to see whether it's necessary and what other restrictions may apply.